app/javascript/app/components/charts/line/line-component.jsx

```diff
@@ -216,13 +217,15 @@ ChartLine.propTypes = {
   forceFixedFormatDecimals: PropTypes.number,
   margin: PropTypes.object,
   domain: PropTypes.object,
-  espGraph: PropTypes.bool.isRequired
+  espGraph: PropTypes.bool.isRequired,
+  dot: PropTypes.bool
 };
 
 ChartLine.defaultProps = {
   height: '100%',
   onMouseMove: () => {},
-  espGraph: false
+  espGraph: false,
```

Comment:
isn't that `espGraph` setting the y and x axis labels? Maybe we could rename it to sth more meaningful, like `showAxisSymbols`

Reply:
I'm going to skip it for now, because it's not very clear to me what else this flag `espGraph` does